src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -169,6 +173,112 @@ void goto_analyzer_parse_optionst::get_command_line_options(optionst &options)
   if(cmdline.isset("error-label"))
     options.set_option("error-label", cmdline.get_values("error-label"));
   #endif
+
+
+  // Output format choice
+  options.set_option("text", false);
+  options.set_option("json", false);
+  options.set_option("xml", false);
+  options.set_option("dot", false);
+  options.set_option("outfile", "-");
+
+  if(cmdline.isset("text"))
+  {
+    options.set_option("text", true);
+    options.set_option("outfile", cmdline.get_value("text"));
+  }
+  else if(cmdline.isset("json"))
+  {
+    options.set_option("json", true);
+    options.set_option("outfile", cmdline.get_value("json"));
+  }
+  else if(cmdline.isset("xml"))
+  {
+    options.set_option("xml", true);
+    options.set_option("outfile", cmdline.get_value("xml"));
+  }
+  else if (cmdline.isset("dot"))
+  {
+    options.set_option("dot", true);
+    options.set_option("outfile", cmdline.get_value("dot"));
+  }
+  else
+  {
+    options.set_option("text", true);
+  }
+
+
+  // Task options
+  options.set_option(    "show", false);
+  options.set_option(  "verify", false);
+  options.set_option("simplify", false);
+
+  if(cmdline.isset("show") ||
+      cmdline.isset("show-intervals") ||
+      cmdline.isset("show-non-null"))
+    options.set_option("show", true);
+  else if(cmdline.isset("verify"))
+    options.set_option("verify", true);
+  else if(cmdline.isset("simplify"))
+  {
+    options.set_option("simplify", true);
+    options.set_option("outfile", cmdline.get_value("simplify"));
+  }
+
+  if(!(options.get_bool_option("show")  ||
+    options.get_bool_option("verify")  ||
+    options.get_bool_option("simplify")))
+  {
+    status() << "Task defaults to --show" << eom;
```

Comment:
The message would be printed before the goto-analyzer banner which should be the first thing.